test/client/connection.spec.js

```diff
@@ -358,29 +358,11 @@ describe('Connection', () => {
             expect(xmlString).to.equal(externalInstanceXML);
         });
 
-        it('fails to load when external data is missing', async () => {
-            resolveExternalInstance = false;
-
-            /** @type {Error | null} */
-            let caught = null;
-
-            try {
-                await connection.getFormParts({
-                    enketoId,
-                });
-            } catch (error) {
-                caught = error;
-            }
-
-            expect(caught instanceof Error).to.equal(true);
-        });
-
         it('loads in preview mode when external data is missing', async () => {
             resolveExternalInstance = false;
```

Comment:
Note, the `resolveExternalInstance` variable and logic is now not used any more. I figured it may be useful to keep it for future tests so kept it in.